docs/concepts/storage/persistent-volumes.md

```diff
@@ -695,6 +695,8 @@ parameters:
 
 #### Azure Disk
 
+##### Azure Unmanaged Disk Storage Class (before v1.7.2)
```

Comment:
Kubernetes users expect a single storage abstraction and ubiquitous access to the storage platform, but Azure Managed Disks dependencies are complex (VMs, premium/non-premium storage, etc). To that end, for Azure Container Service customers we're considering specifying the following scenarios:

- Azure Premium Managed VM can only attach managed disk (both Premium and Standard LRS)
- Azure Managed VM can only attach managed disk (only Standard LRS)
- Unmanaged VM can only attach blob storage disk

For these docs, perhaps it's worth at least mentioning that hybrid clusters with managed disk/blob storage are not recommended. @jdumars, @lachie83, Andy, and I have a task to explore using Azure Instance Metadata Service to apply disk labels to nodes to help with scheduling decisions, but obviously that's not yet an option.